m4_sessions/1903_inning/social_media.md

```diff
@@ -1,56 +1,101 @@
-* Jump right into Black Mirror episode Nose Dive
-* Cut video short so we could have 20-30 minutes of discussion 
+How Connected Are You Really?
+
+
 
 ###### Outcomes
-To make students realize the psychological and sociological impacts of social media on themselves and others.
+To make students realize the (+) & (-) psychological and sociological impacts of social media on themselves and others.
 
 ##### Group Members
-* Marissa Reinke
-* Jon W.
-* Mickey Fen
-* Spencer Carter
+* Michael Clampett
+* Shannon Moranetz
+* Travis Gee
+* Maddie Jones
 
 ## SESSION OUTCOMES
 
 * Realization of implications of social media on yourself and others
 
 * Making people think about their social presence (+/-)
 
-* Is Social Media a direct representation of who you are
+* What are you presenting to the world? How does it compare to your looking glass self?
 
 ## Prompt questions
 
 * Government and Social Media - thoughts/ideas
 
-* What does that mean as a developer and our social responsibility 
+* What does that mean as a developer and our social responsibility? We are the people that make those decisions.
+
+* Why do you use it?
+
+* What apps do you use?
 
-* How do you use it
+* Is there a future for social media without the negative effects listed here? How can that balance be achieved?
 
-* What do you think of double posting? 
 
-* What do you think about people who don't use it? 
 
 ## GROUPING
   
-  * Open floor discussion
+  * Open floor debate-moderated
 
 ## LOGISTICS
 
 | Set Up | Materials Needed | PPT Link (if PPT)|
 | ------ | ---------------- | ---------------- |
-| Room setup in a horseshoe. TVs with speakers | students need: notebooks and something to write with | netflix.com |
+| Large room needed. Argument Posters on both sides of room | students need: open-minded and respectful |
 
 ## SESSION OUTLINE
 
 | Timing | Facilitator Talking Points | Participant Actions  |
-5 min intro / warmup - Jon
-30 min video - TV
-5 min write about why you use social media/how often/could you go without it - Marissa
-15 min discussion time - Spencer
-5 min wrapup  |  What do you participate in currently? OMG how any likes? Then ask again at closing. | Computers closed, minds open, eyes watching tv, ready to talk! Everyone being respectful and waiting their turn to talk. - Fenwick
+larger discussion - need bigger room for activity
+
+10 min Come together - Introduction & Icebreaker Game
+
+tennis ball game - something interesting about yourself
+
+-what brought you to turing, where are you from, etc
+
+15 min Breakout groups  - far enough away that groups can't hear the opposing arguments
+
+-two groups for a Pro
+
+-two groups for a Con
+
+-each of us four will facilitate the discussion
+
+POMODORO - 5 min
+
+15 min breakout groups again - everyone switches sides
+
+POMODORO - 5 min
+
+20 min remaining here.
+
+videos - 10 min.
+
+-Pros - 5 min
+
+-Video - how does social media positively affect humankind?
+
+-autism and interconnectedness
```

Comment:
Would love to see you have a "facilitator's guide" for talking points on the pro/con